docs/releasenotes.asciidoc

```diff
@@ -1,35 +1,25 @@
 [[releasenotes]]
-== Kibana 4.2 Release Notes
+== Kibana 4.3 Release Notes
 
-* Starting with the 2.0 release of Elasticsearch, you can https://github.com/elastic/elasticsearch/pull/9670[no longer] 
-reference fields by leafnode. For example, the `geoip.country_code` field can no longer support searches such as 
-`country_code:US`. Instead, use the entire field name. This change affects saved searches, visualizations, and dashboards 
-from previous versions of Kibana.
+Time-based index patterns are *deprecated* in this release of Kibana. Support for time-based index patterns will be removed
+entirely in the next major Kibana release. Elasticsearch 2.0 includes sophisticated date parsing APIs that Kibana uses to 
+determine date information, removing the need to specify dates in the index pattern name.
 
 [float]
 [[enhancementss]]
 == Enhancements
 
-* {k4issue}2855[Issue 2855]: Configurable Kibana log levels.
-* {k4issue}4204[Issue 4204]: Plugin manager for Kibana server.
-* {k4issue}4218[Issue 4218]: New interface for switching between plugins.
-* {k4issue}3270[Issue 3270]: Server status page.
-* {k4issue}2245[Issue 2245]: Customizable visualization descriptions.
-* {k4issue}2906[Issue 2906]: Dark theme added to dashboards.
-* {k4issue}1902[Issue 1902]: Tile maps support configurable WMS-compliant map servers.
-* {k4issue}2245[Issue 2245]: Customize legends for the Filter aggregation.
-* {k4issue}4817[Issue 4817]: Advanced Settings option maps values to colors. 
-* {k4issue}2874[Issue 2874]: Advanced Settings option to change default timepicker value.
-* {k4issue}2760[Issue 2760]: Adds ability to filter by clicking on a legend. 
-* {k4issue}485[Issue 485]: Adds color rationalization across visualizations.
+* {k4issue}5109[Issue 5109]: Adds custom JSON to visualization filters.
```

Comment:
should this be 1583?  maybe wording can be changed to say it applies to all filters?